components/Search/Search.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+
+import { connect } from 'react-redux';
+import { REDUCER_KEY } from '../../reducers';
+import { getSearchResult } from '../../actions';
+
+import SearchForm from './SearchForm';
+import SearchResult from './SearchResult';
+
+const Search = ({ dispatchSearch, data, loading, done }) => (
+  <div>
+    <SearchForm
+      hintText="Search by name or SteamID" // TODO replace hardcoded with tooltips.json from yasp core
+      disabled={loading}
+      onSubmit={(value) => dispatchSearch(value)}
+    />
+    {done === true ? <SearchResult players={data} /> : ''}
```

Comment:
Is search always players result? If not, we should rename to something ambiguous instead of players.